browser.js

```diff
@@ -107,6 +107,28 @@ define(function(require, exports, module) {
             
             iframe.src = url;
         }
+
+        function setTitle(session, tab, path) {
+            // remember default title between reloads
+            if (!session.doc.title) {
```

Comment:
would something like 
```js
doc.defaultTitle =  "[B] " + path;
tab.title = doc.title || doc.defaultTitle
```
work for your use case?
Current defaultTitle  is a bit misleading, since it's name implies that it is a title, but in reality it is a boolean.